lib/types/luks.nix

```diff
@@ -130,7 +130,7 @@ in
     _create = diskoLib.mkCreateOption {
       inherit config options;
       default = ''
-        if ! blkid "${config.device}" >/dev/null || ! (blkid "${config.device}" -o export | grep -q '^TYPE='); then
```

Comment:
This seems to break stuff:

vm-test-run-disko-luks-btrfs-raid> machine # Device p1 already exists.

Reply:
yeah I'll add a check since the script is run twice.

Reply:
Added in a8ab39de646794b1fa6c71b2dd16df93a3a375b9